crypto/s2n_hmac.c

```diff
@@ -297,8 +297,8 @@ int s2n_hmac_digest_two_compression_rounds(struct s2n_hmac_state *state, void *o
      *
      * 17 bytes if the block size is 128.
      */
-    const uint8_t space_left = (state->hash_block_size == 128) ? 17 : 9;
-    if (state->currently_in_hash_block > (state->hash_block_size - space_left)) {
+    const uint8_t space_left = (state->hash_block_size == 128) ? 111 : 119;
+    if (state->currently_in_hash_block > space_left) {
```

Comment:
This is a nice change but I am worried about `space_left` not being the best name for it now.

Reply:
I reverted this changes because it'd break timing constraints in SideTrail proof.